mkt/extensions/models.py

```diff
@@ -160,19 +159,33 @@ def is_public(self):
         return self.status == STATUS_PUBLIC
 
     @property
-    def manifest_url(self):
-        return absolutify(reverse('extension.manifest',
+    def mini_manifest_url(self):
+        return absolutify(reverse('extension.mini_manifest',
                                   kwargs={'uuid': self.uuid}))
 
     @property
     def mini_manifest(self):
-        """Mini-manifest used for install/update in dict form.
-
-        Actually a maxi-manifest for the moment. Must contain `package_path`,
-        that points to the extension download absolute URL."""
-
-        mini_manifest = deepcopy(self.manifest)
-        mini_manifest['package_path'] = self.download_url
+        """Mini-manifest used for install/update on FxOS devices, in dict form.
+
+        It follows the Mozilla App Manifest format, *not* the Web Extension
+        manifest format.
```

Comment:
I wouldn't mind it if you expounded on this: how it's translated and why this has to be the case. Spending some time thinking on it, I understand it, but it's not very obvious.